src/dev-app/checkbox/checkbox-demo.ts

```diff
@@ -46,7 +47,7 @@ export class AnimationsNoop {
       margin-bottom: 4px;
     }
   `],
-  templateUrl: 'nested-checklist.html',
+  templateUrl: './nested-checklist.html',
```

Comment:
why is that needed? (just asking out of curiosity; anything that changed in ngtsc that needs this?)

Reply:
I'm not really sure why, but without this, WebStorm was unable to resolve the class functions and fields in the template.

It only seemed to be a problem with this one. If it's problematic, I can revert this.